src/cli/commands/start.ts

```diff
@@ -179,6 +201,8 @@ export async function start(outputLocationOtHttpUrl: string, options: SWACLIConf
   // use any specific workflow config that the user might provide undef ".github/workflows/"
   // Note: CLI args will take precedence over workflow config
   try {
+    // TODO: not sure if we should still do this here, as config/user options should override
+    // over any options in the workflow config, but it seems to do the opposite here.
```

Comment:
`readWorkflowFile` does a mixing of the user's options (CLI flags) and the local GitHub action (if available). This was done prior to the info of the user config. Feel free to refactor this section if needed.

Reply:
Once everything else is fixed, I'll try loading the workflow config before everything else set by user (cli args, options...) in option.ts, this should be the most logical option and would also benefit deploy (if nothing else is set).